chopper/lib/src/base.dart

```diff
@@ -367,11 +361,11 @@ class ChopperClient {
   Future<Response<BodyType>> post<BodyType, InnerType>(
     String url, {
     dynamic body,
-    List<PartValue> parts,
-    Map<String, String> headers,
-    Map<String, dynamic> parameters,
-    bool multipart,
-    String baseUrl,
+    List<PartValue>? parts,
+    Map<String, String>? headers,
+    Map<String, dynamic>? parameters,
+    bool? multipart,
```

Comment:
we should avoid nullable boolean, and use `false` as default value

Reply:
Fixed